### DISCUSSION TITLE
Chatfilter suggestion

### DISCUSSION THREAD

**Discussion by jurajplayercze** [Zrips/CMI#6386] [Feature Requests]
Hey, i would appreciate option in chatfilter for caps. 
When player writes something with capslock, it will block the message
![image](https://user-images.githubusercontent.com/83863655/143416889-761a5f67-efc0-40b6-b25d-115d9e6bc4a1.png)

**Comment by ayushgameretawah** [comment]
i want to show cmi ranks in tab or chat both so how to do that please tell bro

**Reply by jurajplayercze** [reply]
Don't know why do you write it here. If you don't know it, you should read CMI wiki, and read their whole config, so you learn everything about it. Learning is better instead repetetive asking

**Reply by mrfloris** [reply]
> i want to show cmi ranks in tab or chat both so how to do that please tell bro

**First and final warning** ayushgameretawah, please stop bothering others with off topic comments. Stick to the topic or make your own discussion. Posting the same question over and over is considered spamming and will be dealt with accordingly.
